src/pages/components/TextInputRef/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { forwardRef } from 'react';
+
+import TextInput, { TextInputProps } from '@/components/TextInput';
+
+const TextInputWithRef = forwardRef<HTMLInputElement, TextInputProps>(
```

Comment:
add a comment above the component //add ref to prevent ref error

Reply:
comment added